### DISCUSSION TITLE
Is it possible to map Classification Store Batch with Excel?

### DISCUSSION THREAD

**Discussion by AlephLau** [pimcore/pimcore#16498] [Q&A / Support]
Hello pimcorenauts!, Is it possible to map Classification Store Batch with Excel? In the documentation, there are examples for CVS files and JSON, but I will receive the information in Excel documents in which I need to import attributes of the classification store type in Quantity values like 6 GB, for example. https://pimcore.com/docs/platform/Data_Importer/Configuration/Mapping_Configuration/Data_Target/Classification_Store_Batch_Details/

**Comment by AlephLau** [comment]
@dpfaffenbauer @fabfischer  PimcoreNautsGods Can you help me with this pls?

**Reply by AlephLau** [reply]
@mattamon Could you help me with this pls?

**Reply by mattamon** [reply]
Hey @AlephCode 
https://pimcore.com/docs/platform/Data_Importer/Configuration/File_Formats the file formats would suggest so yes. You could simply it in our public demo. https://demo.pimcore.com/ -> https://pimcore.com/de/demo for username and pw
